frame/bounties/src/lib.rs

```diff
@@ -151,6 +176,10 @@ pub struct Bounty<AccountId, Balance, BlockNumber> {
 	bond: Balance,
 	/// The status of this bounty.
 	status: BountyStatus<AccountId, BlockNumber>,
+	/// Subbounties counter
+	subbountycount: BountyIndex,
+	/// Sorted list of active Subbounties
+	activesubbounty: Vec<BountyIndex>,
```

Comment:
```suggestion
	active_subbounties: Vec<BountyIndex>,
```

Reply:
I am not sure if we actually need to have this, `active_subbounty_count` should be enough

Reply:
Right now its used for tracking the active subbounty list, and gets used in parent bounty
api 

| api | usage |
| --- | --- |
| award_bounty() | Used to ensure there are no active subbounties before awarding |
| close_bounty() | Used to make iteartive call 
to impl_close_subbounty() [for closing subbounty] |
| extend_bounty_expiry() | Used to make iteartive call 
to impl_extend_subbounty_expiry() [for extending the subbounty expiry] |

May be its good to revisit above requirement before removing this.

Reply:
award_bounty: just use `active_subbounty_count`
close_bounty: you can iterate the SubBounties storage
extend_bounty_expiry: just don't do anything